src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/layout/HmrcStandardLayoutParams.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.viewmodels.layout
+
+import uk.gov.hmrc.govukfrontend.views.Aliases.BackLink
+
+case class HmrcStandardLayoutParams(
+  service: Service = Service(),
+  banners: Banners = Banners(),
+  customLayout: CustomLayout = CustomLayout(),
+  pageTitle: Option[String] = None,
+  backLink: Option[BackLink] = None,
+  cspNonce: Option[String] = None
```

Comment:
the names for the bits of this view model are quite generic, because we have a convention of [hoisting stuff into the Aliases package](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/Aliases.scala#LL19) so that people can wildcard import all the viewmodels - I wonder if having Banners / Service / CustomLayout will cause an issue with that convention or having them generic makes collisions more likely. Though equally I don't like the idea of having long shared prefix "HmrcStandardLayoutParamsService" - the way that aliases stuff works feels like another ticket that might be good to play to see if we can simplify it 🤷‍♂️

Reply:
I'm now wondering if I should ditch this `HmrcStandardLayoutParams` and just pass its contents as parameters to the new template.  Not sure it's adding much, other than more boilerplate in the code & tests.

Reply:
Agree that `Service` and `Banners` are really generic...  and actually, on closer inspection `serviceName` is intended for use when you have a dynamic service name depending on context - by default the service name just comes from messages `service.name`.  So not sure it belongs here, should perhaps be `dynamicServiceName` alongside `pageTitle` etc.?

Reply:
Maybe should have a container for `ServiceURLs` but leave the other stuff out?

Reply:
have replaced `Service` with `ServiceURLs`, with the non-URL fields just pulled up to the top level `...Params`